### ISSUE TITLE
The first option of a database keeps being selected

### ISSUE THREAD

**Issue by Thunder33345** [keepassium/KeePassium#275]
**Description**
When you open a database, the first option of the database is selected(the first group's first entry)

There's no way to back out, if you try to exit selecting said entry, via slide out or the back button, the app will reselect the first entry

There's no way to select another entry because of that

**How to reproduce**
1. Open the app
2. Select a database
3. Observe the behavior when trying to back out of selecting said entry

**Expected behavior**
The app should let you exit selecting said entry

**Environment:**
 - Device: iPhone 7 Plus
 - OS: iOS 14.3
 - App Version: 1.39.122 beta

**Additional context**
Sorry I don't really know how to reproduce it just "happened"

**Comment by keepassium** [comment]
Thanks! Two questions:
1. Does this happen in portrait or landscape mode?
2. Do you observe this consistently every time?

**Comment by Thunder33345** [comment]
1. This happens in portrait mode, which is what I originally described
1.1 When the app starts in portrait and switched to landscape, i could switch to landscape to select another entry in the same folder, but i cannot back out of the folder, search works, so it's an adequate workaround for now
1.2 When the app starts in landscape, it seems to properly select the search bar after opening the DB, behaving as expected, BUT if I select a folder, it seems to follow the behavior as explained(forcefully navigating into the first sub folder, then the first entry)

3. If it's portrait, yes this always happens, regardless of DB selected
3.1 If it's in landscape, as explained in 1.2, selecting a folder with a subfolder triggers it, if I select a folder that has no subfolders inside it, that seems to allow me to back out to the main list

**Comment by keepassium** [comment]
Thank you for the details. I am on the road at the moment, so I will get back to you in about a week.

**Comment by keepassium** [comment]
Hmm, I could not reproduce this on iOS 14.3 emulator or iOS 15.7 device. There are no other reports, but according to Apple, there are only two devices with iOS 14.3 running KeePassium... So this is either some system-specific quirk or maybe even a hardware issue with the touchscreen?

Would you mind updating to the latest available iOS, so we are testing from the same baseline? (For iPhone 7 this would be 15.7.2.)

**Comment by Thunder33345** [comment]
> So this is either some system-specific quirk or maybe even a hardware issue with the touchscreen?

Good point, I will have to do some future testing, I do not believe it's a hardware fault because I never experienced unexpected touches outside of this specific app, if it ware to be a hardware issue, it would happen indiscriminately on all apps

I have a suspicion that it might been a system quirk, I forgot to mention I am running unc0ver jailbreak(hence I really prefer not to upgrade)

I tried to test it without jailbreak active, but seems like there's no differences. As a last resort I decided to delete the app and reinstall it.. but seems like that also does not help.

This seems like a dead end so I don't think there's much we can do here, I don't mind if this gets closed as is.
